src/main/java/org/minions/devfund/alejandro/movie/database/Movie.java

```diff
@@ -0,0 +1,78 @@
+package org.minions.devfund.alejandro.movie.database;
+
+import java.util.ArrayList;
+
+/**
+ * This class contains the information about the movies.
+ */
+public class Movie {
+    private String name;
+    private ArrayList<Actor> actors;
+    private double rating;
+
+    /**
+     * This method is the constructor for movie class.
+     * @param name The name of the movie.
+     */
+    public Movie(final String name) {
+        this.name = name;
+        this.actors = new ArrayList<Actor>();
+    }
+
+    /**
+     * This method is the constructor for movie class.
+     */
+    public Movie() {
+        this.name = " ";
+        this.actors = new ArrayList<Actor>();
+    }
+
+    /**
+     * This method return the name of the movie.
+     * @return the name of the movie.
+     */
+    public String getName() {
+        return this.name;
+    }
+
+    /**
+     * This method will return the list of actors that perform in the movie.
+     * @return an ArrayList.
+     */
+    public ArrayList<Actor> getActors() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)